Custom-Video-Driver/Lets-Build-OTPublisher/Single-Cam-Capturer/TBExampleVideoCapture.m

```diff
@@ -333,24 +395,24 @@ - (NSString*)captureSessionPreset {
 }
 
 - (void) setCaptureSessionPreset:(NSString*)preset {
-    dispatch_async(_capture_queue, ^{
+    dispatch_async([TBExampleVideoCaptureShared captureQueue], ^{
         AVCaptureSession *session = [self captureSession];
         
         if ([session canSetSessionPreset:preset] &&
             ![preset isEqualToString:session.sessionPreset]) {
             
-            [self.captureSession beginConfiguration];
-            self.captureSession.sessionPreset = preset;
-            self->_capturePreset = preset;
+            [_captureSession beginConfiguration];
+            _captureSession.sessionPreset = preset;
+            _capturePreset = preset;
```

Comment:
Nit. Does this make any difference? I understand that self is implicitly captured anyway.